app/models/season.rb

```diff
@@ -1,8 +1,17 @@
 class Season < ActiveRecord::Base
+  include SeasonPhaseSwitcher
+
   validates :name, presence: true, uniqueness: true
 
   before_validation :set_application_dates
 
+  SUMMER_OPEN   = '7, 1'
```

Comment:
Pulling the default dates into a constant is a very good thing! :+1: 

I don't believe you can call `Time.utc` with a string parameter (see line 30). You probably want to use an array with 2 integers instead. But even then you'll have to splat the argument call like this:

```ruby
Time.utc(year, *SUMMER_OPEN) # 1 jul
```

A comment on what the values for the constants mean (= month, day) would be good for new devs or future versions of ourselves.

A different approach could be assigning Procs or Lambdas to the constants but that may get a little too verbose.